components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -182,23 +195,34 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
     }));
   }
 
-  updateTeam(): void {
-    const newTeam = <Team>Object.assign(
-      {}, this.team, { name: this.editForm.get('teamName').value });
-    this.store.dispatch(new UpdateTeam(newTeam));
+  public handleNameChange(): void {
+    this.disableSave = this.updateNameForm.controls['name'].value === this.team.name;
```

Comment:
I'd prefer we used the formgroup's means to determine if this is dirty or not instead of re-creating that logic using an instance variable.